pycparser/c_parser.py

```diff
@@ -1580,6 +1580,14 @@ def p_labeled_statement_3(self, p):
         """ labeled_statement : DEFAULT COLON pragmacomp_or_statement """
         p[0] = c_ast.Default([p[3]], self._token_coord(p, 1))
 
+    def p_labeled_statement_4(self, p):
+        """ labeled_statement : ID COLON SEMI"""
```

Comment:
I don't think this rule (_4) should be necessary, since a statement can already be empty. Why do you need it?